### ISSUE TITLE
Consistent ordering & presentation of breadcrumbs & detail tabs 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [tamzi/bungeni-portal#566]
```
It is possible in bungeni to drill-down to a same object along different
paths i.e. to see the same object within different contexts. Depending on
the context, a number of detail tabs may be shown to the user. 

Commonly a subset of these detail tabs is shown in multiple contexts.
However, currently the order of these detail tabs changes for no apparent
reason between contexts. 

Example (using a fresh test setup) -- drill-down to "Stella MP33" in two ways:

first via "members > current > stella..." shows:
member of parliament | personal info | Offices held | Parliamentary activities 

second via "business > committees > ECPW... > membership > stella..." shows:
member | Offices held | personal info

"Offices held" and "personal info" are switched, adding unnecessary effort
for the user to understand what is going on.

Plus, the more contextual detail tabs should not be spliced into the list
of tabs arbitrarily, but always at the beginning of the list i.e. the
"parliamentary activties" being very contextual, should appear before the
more generic tabs. 

Plus, the 2nd "member" tab label should also be made more clear that this
is the "member of the ECPW committee".

This is just a single example -- the entire portal UI should be audited and
made consistent as suggested here.



```

Original issue reported on code.google.com by `mario.ruggier@gmail.com` on 9 Mar 2010 at 3:17
